lib/matplotlib/ticker.py

```diff
@@ -1001,12 +1001,14 @@ def __call__(self, x, pos=None):
             return '0'
 
         x = abs(x)
+        if x == np.inf or x == np.nan:
```

Comment:
using `np.isfinite` here is probably better, `==` with anything float-like is usually not a good idea.

Reply:
Note in particular
![image](https://github.com/user-attachments/assets/ee87add0-b209-4b9e-abe8-b5ca600115e5)

Reply:
> Note in particular

Thanks for the reminder! I somehow missed @tacaswell's comments from two weeks ago. (probably they got lost between emails from `#28495`...).